pages/goals/new-goal.js

```diff
@@ -1,6 +1,8 @@
-import NewGoalForm from '../components/goals/NewGoalForm';
-import Page from '../components/Page';
-import { useSession, signIn } from 'next-auth/react';
+// @mui
+import { useSession, signIn, signOut } from 'next-auth/react';
```

Comment:
Is it possible to create only once for everyplace this part of code ==> import { useSession, signIn, signOut } from 'next-auth/react';